src/model/account/Account.ts

```diff
@@ -78,6 +78,31 @@ export class Account {
         );
     }
 
+    /**
+    * Create an Account from a given mnemonic string
+    * @param mnemonic - The mnemonic string
+    * @param networkType - Network type
+    * @param {SignSchema} signSchema The Sign Schema. (KECCAK_REVERSED_KEY / SHA3)
+    * @return {Account}
+    */
+    public static createFromMnemonic(mnemonic: string,
+        networkType: NetworkType,
+        signSchema: SignSchema = SignSchema.SHA3): Account {
+
+        if(!Crypto.isValidMnemonic(mnemonic)) {
+            throw Error(`Invalid mnemonic: ${mnemonic.length}`);
+        }
+      
+        const hashKey = convert.uint8ToHex(Crypto.mnemonicToSeed(mnemonic));
+      
```

Comment:
You can use just `Account.createFromPrivateKey` static method instead the stuff below.

Reply:
changed to use Account.createFromPrivateKey